pyproject.toml

```diff
@@ -106,7 +106,7 @@ convention = "google"
 ]
 
 [tool.ruff.lint.pylint]
-max-args = 15
+max-args = 20
```

Comment:
let's discuss this - because by doing this you are araison the max for all function. I would rather we keep it at 15 and make an exception in the file where you are getting the issue